src/index.css

```diff
@@ -0,0 +1,39 @@
+
+body{
+    margin: 0 auto;
+  
+}
+.yellow{
+    color: #FCDD06;;
+}
+.logoName
+{
+    /* top: 0px;
+    left: 73px;
+    width: 107px;
+    height: 50px;
+    /* UI Properties */
+    /* text-align: left;
+    font: normal normal 900 40px/15px Muli;
+    letter-spacing: 1.6px; */ 
+    color: #FFFFFF;
+}
+
+.MuiToolbar-root{
+    top: 0px;
+    left: 0px;
+   padding: 0px 70px;
+    height: 85px;
+    /* UI Properties */
+    background: #242424 0% 0% no-repeat padding-box;
+    opacity: 1;
+}
+.css-to4x1n-MuiBadge-badge{
```

Comment:
this class is useless because it's auto-generated by MUI at runtime, Find another way to apply this style.

Reply:
O
kay doctor